lib/em-hiredis/connection_manager.rb

```diff
@@ -12,6 +12,10 @@ module EventMachine::Hiredis
   class ConnectionManager
     include EventEmitter
 
+    DEFAULTS = {
+      :reconnect_attempts => 3
+    }
```

Comment:
I think this should be a top-level variable, e.g.:

```ruby
DEFAULT_RECONNECT_ATTEMPTS = 3
```

Reply:
you mean it should be under the module rather than class?

Reply:
Other "constants" in this library are defined under the class, so I would stick to that.